Eatery/Controllers/BRB/BRBLoginViewController.swift

```diff
@@ -58,28 +59,30 @@ class BRBLoginViewController: UIViewController {
 
             stackView.isUserInteractionEnabled = !isLoading
             loginButton.isEnabled = !isLoading
+
+            tableView.layoutSubviews()
         }
     }
 
+    private var favoriteItems = Defaults[\.favoriteFoods]
+    private let favoriteId = "favorite"
+    private let loginId = "login"
+    var tableView: UITableView!
```

Comment:
Can be 

```suggestion
    var tableView = UITableView(frame: .zero, style: .grouped)
```

If you were starting from scratch I would do it this way, but if we use `!` in eatery then it's better to stay consistent.